### ISSUE TITLE
feature request: view current session ID again

### ISSUE THREAD

**Issue by edmundlaugasson** [gravitational/teleconsole#28]
Currently only in the beginning when you start a session, you will see a session ID and WebUI URL. It would be nice if there could be a way to see current session ID and URL with a command.
E.g.: 
teleconsole --session
teleconsole --id
... or similar.
If I have used a terminal for a while - there might be necessary to check the session ID again in order to share it.
Then there would be possible to pipe it e.g. to pastebinit to get that ID and no need to retype it and avoid mistakes. Currently the most challenging is to type the session ID again because I cannot copy it: using .e.g virtual machine or just pure console without GUI.

**Comment by kontsevoy** [comment]
@edmundlaugasson good idea. it used to be set in an environment variable in early releases... going to bring this back.
